docs/source/quick_start/resources/init.cast

```diff
@@ -0,0 +1,32 @@
+{"version":2,"width":120,"height":10}
+[0,"o","~ $ "]
+[0.6994607448577881,"o","c"]
+[0.9944193363189697,"o","d"]
+[1.1819167137145996,"o"," "]
+[1.432516098022461,"o","m"]
+[1.6514792442321777,"o","y"]
+[1.8078022003173828,"o","_mcdr_server/"]
+[2.151972532272339,"o","\r\n"]
+[2.1520578861236572,"o","\u001b[?2004l"]
+[2.199392080307007,"o","\u001b[?2004h"]
+[2.214003324508667,"o","my_mcdr_server $ "]
+[2.669487476348877,"o","m"]
+[2.7335927486419678,"o","c"]
+[2.951856851577759,"o","d"]
+[3.106980085372925,"o","reforged"]
+[3.287646484375,"o"," "]
+[3.317957258224487,"o","i"]
+[3.427799320220947,"o","n"]
+[3.522567844390869,"o","it"]
+[3.719502925872803,"o","\r\n"]
+[4.1195611000061035,"o","\u001b[?2004l"]
+[4.239581108093262,"o","Initialized environment for MCDReforged in C:\\Users\\Alex3236\\Documents\\~\\my_mcdr_server\r\n"]
```

Comment:
A more general path will be better, e.g. `D:\my_mcdr_server`. Although I'll prefer linux environment more

Reply:
My WSL seems to be broken by Windows Update so I used git bash
i'll change this to `~/my_mcdr_server`